lametro/templates/index/index.html

```diff
@@ -172,5 +175,47 @@ <h2 style="display: inline-block;">
 
         $('#beta-info').tooltip({html: true, trigger: 'focus click', placement: 'right'});
 
+        function toggleAlertManager(){
+            $("#alert-manager").toggle();
+        }
+
+        toggleAlertManager();
+
+        $("#alert-manager-toggle").click(function() {
+            toggleAlertManager();
+            if ($(this).text() == "Show"){
+                $(this).text("Hide")
+            } else {
+                $(this).text("Show")
+            }
+            return false;
+        });
+
+        $("#id_type").on('change', function() {
+            $(this).removeClass()
+            $("#id_description").removeClass()
+            if (this.value != "") {
+                $(this).addClass("alert-" + this.value)
+                $("#id_description").addClass("alert-" + this.value)
+            }
+        });
+
+
+        $("#link, #bold, #italic").on('click', function(){
+            input = document.getElementById("id_description")
+
+            switch(this.id) {
+                case "link":
+                    input.value += ("[this is a link!](https://url.com) ")
+                    break;
+                case "bold":
+                    input.value += ("**this is bolded text!** ")
+                    break;
+                case "italic":
+                    input.value += ("*this is italicized text!* ")
+                    break;
```

Comment:
I'm torn on whether to keep this bit here, what do you think of this?

I initially wanted something like github where you can highlight some text, and clicking one of the buttons shown below would wrap your selection in markdown, but couldn't get `window.getSelection()` to work on firefox. So scrapped that approach in favor of just having the buttons insert snippets of markdown that the user could edit to what they need.

Is this useful enough to keep the way it is now? I feel like I'm leaning towards either getting rid of that feature all together, or figuring out how to get it to work the way I imagined in the first place, but not sure.

![image](https://github.com/Metro-Records/la-metro-councilmatic/assets/114717958/ae40e07a-07f1-4eba-b823-d24e15e13670)

Reply:
I think it's a little idiosyncratic, but ultimately more ergonomic than their having to create the Markdown from scratch. I vote keep, though this is also a great question for the user.